### PR TITLE
Mon 12695 pb ssh connector

### PR DESCRIPTION
**Pull request by jean-christophe81**

### CODE REVIEW

File: packaging/rpm/centreon-collect.spec [hunk 1]
```diff
@@ -137,7 +137,6 @@ a monitoring engine.
 Summary: Centreon Connector SSH provide persistante connection between checks.
 Group: Application/System
 Requires: centreon-clib = %{version}-%{release}
-Requires: libssh2 >= 1.4
 Requires: libgcrypt
```
Comment (bouda1) [inline packaging/rpm/centreon-collect.spec:140]: Do you think libgcrypt is still needed now?

File: tests/connector_ssh/conf_engine/contactgroups.cfg [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+#                   - Julien Mathis                               #
+#                   - Romain Le Merlus                            #
+#                                                                 #
+#                           www.centreon.com                      #
+#                For information : contact@centreon.com           #
+###################################################################
+#                                                                 #
+#         Last modification 2022-03-31 16:04                      #
+#         By admin_admin                                          #
+#                                                                 #
```
Comment (bouda1) [inline tests/connector_ssh/conf_engine/contactgroups.cfg:15]: if you don't use this file, you just have to remove it and remove its call in centengine.cfg

File: tests/connector_ssh/conf_engine/dependencies.cfg [hunk 3]
```diff
@@ -0,0 +1,16 @@
+###################################################################
```
Comment (bouda1) [inline tests/connector_ssh/conf_engine/dependencies.cfg:1]: again here

File: tests/connector_ssh/conf_engine/escalations.cfg [hunk 4]
```diff
@@ -0,0 +1,16 @@
+###################################################################
```
Comment (bouda1) [inline tests/connector_ssh/conf_engine/escalations.cfg:1]: here too

File: tests/connector_ssh/conf_engine/hostgroups.cfg [hunk 5]
```diff
@@ -0,0 +1,17 @@
+###################################################################
+#                                                                 #
```
Comment (bouda1) [inline tests/connector_ssh/conf_engine/hostgroups.cfg:2]: the same here

File: tests/connector_ssh/conf_engine/meta_commands.cfg [hunk 6]
```diff
@@ -0,0 +1,16 @@
+###################################################################
```
Comment (bouda1) [inline tests/connector_ssh/conf_engine/meta_commands.cfg:1]: the same here

File: tests/connector_ssh/conf_engine/meta_host.cfg [hunk 7]
```diff
@@ -0,0 +1,16 @@
+###################################################################
```
Comment (bouda1) [inline tests/connector_ssh/conf_engine/meta_host.cfg:1]: the same here

File: tests/connector_ssh/conf_engine/meta_services.cfg [hunk 8]
```diff
@@ -0,0 +1,16 @@
+###################################################################
```
Comment (bouda1) [inline tests/connector_ssh/conf_engine/meta_services.cfg:1]: the same here

File: tests/connector_ssh/conf_engine/meta_timeperiod.cfg [hunk 9]
```diff
@@ -0,0 +1,16 @@
+###################################################################
```
Comment (bouda1) [inline tests/connector_ssh/conf_engine/meta_timeperiod.cfg:1]: the same here

File: tests/connector_ssh/conf_engine/servicegroups.cfg [hunk 10]
```diff
@@ -0,0 +1,17 @@
+###################################################################
```
Comment (bouda1) [inline tests/connector_ssh/conf_engine/servicegroups.cfg:1]: the same here

**Review by bouda1 — APPROVED** [review]

File: connectors/ssh/CMakeLists.txt [hunk 11]
```diff
@@ -31,37 +31,35 @@ add_executable(centreon_connector_ssh
...
 )
 add_dependencies(centreon_connector_ssh centreon_clib)
-target_link_libraries(centreon_connector_ssh ${LIBSSH2_LIBRARIES}
-  centreon_clib ${LIBGCRYPT_LIBRARIES} CONAN_PKG::spdlog CONAN_PKG::fmt pthread)
+target_link_libraries(centreon_connector_ssh 
+  centreon_clib ${LIBGCRYPT_LIBRARIES} CONAN_PKG::spdlog CONAN_PKG::fmt CONAN_PKG::libssh2 pthread)
+#target_link_libraries(centreon_connector_ssh ${Libssh2_LIBRARIES}
+#  ${CLIB_LIBRARIES} ${LIBGCRYPT_LIBRARIES} ${spdlog_LIBS} ${fmt_LIBS} pthread)
+
+
```
[the remark this answers is not in the document]
Comment (jean-christophe81) [reply connectors/ssh/CMakeLists.txt]: done

**Pull request merged**